xbmc/pvr/PVRManager.cpp

```diff
@@ -1414,6 +1414,28 @@ bool CPVRManager::IsIdle(void) const
   return true;
 }
 
+bool CPVRManager::IsSystemPowerdownOkay(bool bAskUser /*= true*/) const
```

Comment:
@xhaggi: Good point. I don't like the name either, but had no better idea. Will change the method's name to CanSystemPowerdown.